cloudsplaining/command/scan_multi_account.py

```diff
@@ -0,0 +1,205 @@
+"""Scan multiple AWS accounts via AssumeRole"""
+import logging
+import os
+import json
+import yaml
+import click
+import boto3
+from click_option_group import optgroup, RequiredMutuallyExclusiveOptionGroup
+from cloudsplaining.shared.constants import EXCLUSIONS_FILE
+from cloudsplaining.command.download import get_account_authorization_details
+from cloudsplaining import set_log_level
+from cloudsplaining.shared.exclusions import Exclusions, DEFAULT_EXCLUSIONS
+from cloudsplaining.shared import utils, aws_login
+from cloudsplaining.shared.validation import check_authorization_details_schema
+from cloudsplaining.scan.authorization_details import AuthorizationDetails
+from cloudsplaining.output.report import HTMLReport
+
+logger = logging.getLogger(__name__)
+
+
+@click.command(
+    short_help="Scan multiple AWS Accounts using a config file"
+)
+@click.option(
+    "--config-file",
+    "-c",
+    type=click.Path(exists=True),
+    required=True,
+    help="Path of the multi-account config file",
+)
+@click.option(
+    "--profile",
+    "-p",
+    type=str,
+    required=False,
+    help="Specify the AWS IAM profile.",
+    envvar="AWS_PROFILE"
+)
+@click.option(
+    "--role-name",
+    "-r",
```

Comment:
🎨  For clarify insert the name of the variable, `role_name`, here. May want to do this for other `click.option`s for consistency.

Reply:
Done